toolchain/BUILD

```diff
@@ -0,0 +1,10 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# A convenience target for running the toolchain with the full prelude
+# available.
+alias(
+    name = "carbon",
+    actual = "//toolchain/install:prefix_root/bin/carbon",
```

Comment:
Does this cause the core libraries to get built and installed into the install prefix? I may just be missing it, but I don't see a suitable dependency in `toolchain/install/BUILD`.

Reply:
╚╡bazel query 'somepath(//toolchain/install:prefix_root/bin/carbon, //toolchain/install:core_data)'
//toolchain/install:prefix_root/bin/carbon
//toolchain/driver:carbon
//toolchain/driver:driver
//toolchain/install:install_lib_data
//toolchain/install:core_data

Reply:
I totally grant that I don't like this kind of circular dependency.